Add first part of DataPager articles. (#987)
telerik/maui-docs: controls/datapager/styling/navigationview.md

---
title: Navigation View Styling
page_title: .NET MAUI DataPager Documentation - NavigationView Styling
description: Learn what are the styling options for the Telerik UI for .NET MAUI DataPager NavigationView view.
position: 2
slug: datapager-styling-navigationview
---

# Styling the .NET MAUI DataPager Navigation View

> The DataPager for .NET MAUI provides a styling mechanism for customizing the look of the navigation view that presents the number of pages and the combo that allows you to select different page, by setting the `NavigationViewStyle` (`Style` with target type of `telerik:DataPagerNavigationView`) property.

Reviewer: ```suggestion
The DataPager for .NET MAUI provides a styling mechanism for customizing the look of the navigation view, which presents the number of pages and the combo box for selecting different pages.
```

> The `NavigationViewStyle` uses as a target type the `telerik:DataPagerNavigationView`. The `DataPagerNavigationView` exposes the following properties:

Reviewer: ```suggestion
To style the Navigation View, utilize the `NavigationViewStyle` property with a target type of `telerik:DataPagerNavigationView`). 

The `DataPagerNavigationView` exposes the following properties:
```

* `DataPager` (`RadDataPager`)&mdash;Specifies the associated `Telerik.Maui.Controls.RadDataPager`.
* `LabelStyle` (`Style` with target type `Label`)&mdash;Specifies the style which applies to the label.
* `ComboBoxStyle` (`Style` with target type `RadComboBox`)&mdash;Specifies the associated `Telerik.Maui.Controls.RadDataPager`.
* `BackgroundColor` (`Color`)&mdash;Specifies the background color of the page sizes view.

> Here is an example.

Reviewer: ```suggestion
Here is an example demonstrating how to use the `NavigationViewStyle` property.
```

> Define the DataPager in XAML:

Reviewer: Use a list for the steps.

```XAML
<telerik:RadDataPager NavigationViewStyle="{StaticResource NavigationViewStyle}">
```

Set the styles to the page's resources:

```XAML
<Style x:Key="ComboStyle" TargetType="telerik:RadComboBox">

</Style>
<Style x:Key="LabelStyle" TargetType="Label">

</Style>
<Style x:Key="NavigationViewStyle" TargetType="telerik:DataPagerNavigationView">
    <Setter Property="ComboBoxStyle" Value="{StaticResource ComboStyle}" />
    <Setter Property="LabelsStyle" Value="{StaticResource LabelStyle}" />
</Style>
```

## Styling the Navigation ComboBox View

> The DataPager for .NET MAUI provides a styling mechanism for customizing the look of the view that allows you to navigate to the pages by using the `NavigationComboBoxStyle` (`Style` with target type of `telerik:RadComboBox`) property.

Reviewer: ```suggestion
To customize the appearance of the ComboBox view that allows the navigation to a specific page, use the `NavigationComboBoxStyle` (`Style` with target type of `telerik:RadComboBox`) property.
```

> The `NavigationComboBoxStyle` uses as a target type the `RadComboBox` control. Use the styling properties described in the [ComboBox Styling]({%slug combobox-styling%}) article to style the combobox part of the DataPager control. 

Reviewer: ```suggestion
The target type of the `NavigationComboBoxStyle` is the `RadComboBox` control. Use the styling properties described in the [ComboBox Styling]({%slug combobox-styling%}) article to style the ComboBox part of the DataPager control. 
```

> Here is an example:

Reviewer: ```suggestion
Here is an example demonstrating how to use the `NavigationComboBoxStyle` property.
```

---
title: Navigation View Styling
page_title: .NET MAUI DataPager Documentation - NavigationView Styling
description: Learn what are the styling options for the Telerik UI for .NET MAUI DataPager NavigationView view.
position: 2
slug: datapager-styling-navigationview
---

# Styling the .NET MAUI DataPager Navigation View

The DataPager for .NET MAUI provides a styling mechanism for customizing the look of the navigation view, which presents the number of pages and the combo box for selecting different pages.

> To style the Navigation View, utilize the `NavigationViewStyle` property with a target type of `telerik:DataPagerNavigationView`). 

Reviewer: ```suggestion
To style the Navigation View, utilize the `NavigationViewStyle` property (of type `Style` with a target type of `telerik:DataPagerNavigationView`). 
```
